S/SCIP_PaPILO/build_tarballs.jl

```diff
@@ -14,6 +14,8 @@ sources = [
 script = raw"""
 cd scipoptsuite*
 
+LDFLAGS="${LDFLAGS} -lgfortran"
```

Comment:
You have to export variables to have any effect on subprocesses
```suggestion
export LDFLAGS="-lgfortran"
```